ch1/study2.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="utf-8" />
+    <title>This is TITLE!!</title>
+  </head>
+  <body>
+    <h1>This is h1</h1>
+    <!--This is comment and h1 is very important!-->
+    <a href=https://www.naver.com>This is hyperLink feat. a tag</a>
+    <h2>This is h2</h2>
+
+    <fieldset>
+      <legend>Legend tag is name</legend>
+      <input />
+      <input type="checkbox" />
+      <input type="radio" />
```

Comment:
```suggestion
      <input type="text">
      <input type="checkbox">
      <input type="radio">
```

이 부분도 마찬가지!
추가적으로 `input` 태그의 경우 `type`을 명시해주는 편입니다.